helm_deploy/values-preprod.yaml

```diff
@@ -17,7 +17,7 @@ ingress:
 
 env:
   JAVA_OPTS: "-Xmx512m"
-  SPRING_PROFILES_ACTIVE: "preprod,secure,logstash"
+  SPRING_PROFILES_ACTIVE: "preprod,logstash"
```

Comment:
I don't think we want to do this permanently ? I usually just change the deployment with kubectl ?

Reply:
Yeah I did consider this but then because there were multiple changes over multiple services(albeit trivial ones) I thought it might be better to have it in version control. TBH I'm happy either way but this prevents us inadvertently overwriting stuff if we do any new deployments which isn't so much of a risk for CPG but more so for court-case-service 🤷‍♂️

Reply:
we will of course have to roll it back after but again it's pretty trivial